### ISSUE TITLE
data consistency between staging and production sites

### ISSUE THREAD

**Issue by hancush** [datamade/la-metro-councilmatic#430]
we import data on a custom schedule to the production database, but only once a day to the staging database. we also delete things from the production database but don't always delete them from staging. these appear to have caused some data drift between databases. for many changes, this is ok, however it makes it difficult to qa changes that are highly dependent on the state of data, like #429.

a relatively quick fix would be to copy the production database to the staging database. this leaves the possibility for the databases to drift out of sync. if this is truly undesirable, we could set up an automated process to sync the database contents on some schedule.

**Comment by hancush** [comment]
@jmithani made a big contribution to this issue by making some data fixes to resolve persistent import issues on staging. i think there's still an issue of removing data from both databases, so i'll leave this open, but it's def lower priority now that data that _should_ exist in both databases actually does. ty jasmine!!!
